packages/webapp/components/layouts/Onboarding/index.module.css

```diff
@@ -0,0 +1,14 @@
+.video {
+  --video-scale: 1.25;
+  transform: scale(var(--video-scale));
+
+  @media (min-height: 800px) and (min-width: 1020px) {
+    --video-scale: 1.5;
+  }
+  @media (min-height: 800px) and (min-width: 1240px) {
+    --video-scale: 1.8;
+  }
+  @media (min-height: 1000px) and (min-width: 1668px) {
+    --video-scale: 2;
+  }
+}
```

Comment:
Since we introduced `laptopH` can't we do this with tailwind?

Reply:
I tried, but it would clash with widths.

Reply:
I think for now use this approach, it's a bit messy because non-designers are requesting changes.
I would say maybe rather use `@apply` to import the correct scaling, this is very susceptible to tailwind changing the variants one day.

So keep the inline media queries, but use `@apply` I think it should work.